_extras/guide.md

```diff
@@ -67,6 +69,7 @@ Instructors should, however, walk learners through some basic operations:
 *   Create a new notebook.
 *   Enter code or data in a cell and execute it.
 *   Explain the difference between `In[#]` and `Out[#]`.
+*   Show that modifying code in an earlier cell requires re-running any subsequent cells in order for it to reflect the change.
```

Comment:
One more thing @ndilekli , we try to keep the lines less than 100 characters in the markdown (see #498 for explanation). Here I am suggesting the change to shorten the line and to change "it" to "them". Thanks again for all your work on this! It's good to know it is teaching you something about collaborating on GitHub.
```suggestion
*   Show that modifying code in an earlier cell requires re-running any subsequent
    cells in order for them to reflect the change.
```

Reply:
Sure thing!